doc/releases/octopus.rst

```diff
@@ -1,3 +1,26 @@
+v15.2.2 Octopus
+===============
+
+This is the first bugfix release of Ceph Octopus, we recommend all Octopus users
+upgrade.
+
+Notable Changes
+---------------
+
+* issue#45589: The name of the rbd pool object that is used to store
+  rbd trash purge schedule is changed from "rbd_trash_trash_purge_schedule"
+  to "rbd_trash_purge_schedule". Users that have already started using
+  ``rbd trash purge schedule`` functionality and have per pool or namespace
+  schedules configured should copy "rbd_trash_trash_purge_schedule"
+  object to "rbd_trash_purge_schedule" before the upgrade and remove
+  "rbd_trash_purge_schedule" using the commands::
```

Comment:
... using the following commands in every RBD pool and namespace where a trash purge schedule was previously configured::